config/jobs/kubernetes/sig-testing/compatibility-versions-e2e.yaml

```diff
@@ -191,7 +191,7 @@ periodics:
     path_alias: k8s.io/test-infra
   spec:
     containers:
-    - image: gcr.io/k8s-staging-test-infra/krte:v20250527-1b2b10e804-master
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20250527-1b2b10e804-master
```

Comment:
These don't always get built at the same tags (since they have a partially overlapping set of inputs) but this image does exist (you can browse to it)